src/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -311,30 +130,28 @@ int run_as_app(
         return StatusCode::Success;
     }
 
+    return StatusCode::InvalidArgFailure;
+}
+
+int run_as_app(
+    hostpolicy_init_t &hostpolicy_init,
+    const arguments_t &args)
+{
+    run_context_t context {};
+    int rc = context.initialize(g_init, args, true /* enable_breadcrumbs */);
+    if (rc != StatusCode::Success)
+        return rc;
+
     // Verbose logging
```

Comment:
You could also move the property tracing to the `create_coreclr` as it's done by both cases.